filepath.go

```diff
@@ -91,24 +91,14 @@ func LineCnt(fileName string) int {
 
 // LineCntr counts no. of lines for given reader
 func LineCntr(r io.Reader) int {
-	buf := make([]byte, 8196)
-	count := 0
-	lineSep := []byte{'\n'}
-
-	for {
-		c, err := r.Read(buf)
-		if err != nil && err != io.EOF {
-			return count
-		}
 
-		count += bytes.Count(buf[:c], lineSep)
+	contents, err := ioutil.ReadAll(r)
```

Comment:
@adelowo Yes, I understanding it will improve performance. However it has huge downside.

You're loading all the file content into memory and counting the newline. It's good for smaller size file not for big one.

For e.g: This function is used in `log` package to count log file lines for file rotation.